sdk/communication/communication-short-codes/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-11-09)
+
+- Test Release Pipeline
```

Comment:
There is a specific format that Changelogs should follow with ### Added / ### Breaking Changes etc.

Let's also write meaningful text here, for inspiration https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/communication/communication-sms/CHANGELOG.md#100-beta1-2020-09-22